### PR TITLE
Add themes to license dialog

### PR DESCRIPTION
**Pull request by TobiGr**
This PR enables themes for the license dialogs.

Addresses #646

### CODE REVIEW

File: app/src/main/assets/gpl_2.html [hunk 1]
```diff
@@ -15,9 +15,9 @@
 </p>
 
 <pre>
-Copyright (C) 1989, 1991 Free Software Foundation, Inc.  
-51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
-
+Copyright (C) 1989, 1991 Free Software Foundation, Inc.<br/>
```
Comment (coffeemakr) [inline app/src/main/assets/gpl_2.html]: Are those line breaks necessary? IMO there is no need to add them since the text is inside a `<pre>`.

**Comment by mauriciocolli** [comment]
Some things I noticed:
- UTF-8 text is not displaying correctly in the webview, and a quick test shows that the file is fetched correctly (check by logging right after it's read). 
    - Test on MPL-2 to see this character `“`.
- I/O in the main thread

File: app/src/main/assets/gpl_2.html (hunk 1 above)
Comment (TobiGr) [reply app/src/main/assets/gpl_2.html]: You are right, but I use `pre { white-space: pre-wrap; }` to keep the width of the body at 100% of the view. That's why I added the line breaks manually.

**Comment by TobiGr** [comment]
> UTF-8 text is not displaying correctly in the webview, and a quick test shows that the file is fetched correctly (check by logging right after it's read).
 
@mauriciocolli Thanks for the stackoverflow link, that works.
I don't have enough time to fix the I/O thing this week, so I am going to do this next week as soon as I can.

**Comment by theScrabi** [comment]
No problem

File: app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+import android.os.AsyncTask;
+import android.support.v7.app.AlertDialog;
+import android.webkit.WebView;
+import org.schabi.newpipe.R;
+import org.schabi.newpipe.util.ThemeHelper;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+public class LicenseFragmentHelper extends AsyncTask<Object, Void, Integer> {
```
Comment (TobiGr) [inline app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.java]: @mauriciocolli Should I make this class an inner class of `LicenseFragment`?

**Comment by theScrabi** [comment]
this does it :)

**Pull request merged**